convert2rhel/data/8/x86_64/repos/oracle-8.4/virt-ol8.repo

```diff
@@ -0,0 +1,6 @@
+[ol8_kvm_appstream]
```

Comment:
Remove this whole file for the reason that it's not OL 8.4-specific. It might or might not require dependencies from OL 8.5+ BaseOS/Appstream.

Reply:
Done.